server_addon/applications/server/settings.py

```diff
@@ -144,7 +144,17 @@ def validate_unique_name(cls, value):
 
 
 class ApplicationsSettings(BaseSettingsModel):
-    """Applications settings"""
+    """Applications settings
+
+    Define applications are available to your studio, locations of their
+    executables, and their additional environments.
+
+    AYON uses the following definitions to populate the applications list
+    in project anatomy > attributes.
+    So, You'd still need modify your project anatomy settings.
+
+    Use shortcut **a+a** to navigate to project anatomy.
```

Comment:
I feel like front-end shortcuts should not be in the docstrings of the codebase - it'll just be too hard to keep in sync.
```suggestion
```